src/test/java/CalendarServletTest.java

```diff
@@ -0,0 +1,140 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import com.google.api.services.calendar.model.Event;
+import com.google.common.collect.ImmutableList;
+import com.google.sps.model.AuthenticationVerifier;
+import com.google.sps.model.CalendarClient;
+import com.google.sps.model.CalendarClientFactory;
+import com.google.sps.servlets.CalendarServlet;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.security.GeneralSecurityException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mockito;
+
+/** Test Calendar Servlet responds to client with correctly parsed Events. */
+@RunWith(JUnit4.class)
+public final class CalendarServletTest {
+  private AuthenticationVerifier authenticationVerifier;
+  private CalendarClientFactory calendarClientFactory;
+  private CalendarClient calendarClient;
+  private CalendarServlet servlet;
+
+  private static final boolean AUTHENTICATION_VERIFIED = true;
+  private static final String ID_TOKEN_KEY = "idToken";
+  private static final String ID_TOKEN_VALUE = "sampleId";
+  private static final String ACCESS_TOKEN_KEY = "accessToken";
+  private static final String ACCESS_TOKEN_VALUE = "sampleAccessToken";
+  private static final Cookie sampleIdTokenCookie = new Cookie(ID_TOKEN_KEY, ID_TOKEN_VALUE);
+  private static final Cookie sampleAccessTokenCookie =
+      new Cookie(ACCESS_TOKEN_KEY, ACCESS_TOKEN_VALUE);
+  private static final Cookie[] validCookies =
+      new Cookie[] {sampleIdTokenCookie, sampleAccessTokenCookie};
+
+  private HttpServletRequest request;
+  private HttpServletResponse response;
+  private StringWriter stringWriter;
+  private PrintWriter printWriter;
```

Comment:
extreme nit: maybe move this up with the AuthenticationVerifier, CalendarClientFactory, etc. just for readability

Reply:
Agreed. Fixed